language/defined_spec.rb

```diff
@@ -900,6 +900,14 @@
     defined?(DefinedSpecs::Undefined).should be_nil
   end
 
+  it "returns nil when the constant is not defined and the parent implements const_missing" do
```

Comment:
The term `parent` is not accurate. It should be about outer scope/module/etc of a constant.

**minor:** it's common to prefix a method name with either `.` or `#` so `const_missing` becomes `. const_missing`.